src/ir.h

```diff
@@ -66,13 +66,17 @@ struct Const {
   struct F64 {};
 
   // Keep the default constructor trivial so it can be used as a union member.
-  Const() = default;
+  Const() {
```

Comment:
I'd prefer picking something valid, like just forwarding this to `Const(I32(), 0)`